.github/dependabot.yml

```diff
@@ -0,0 +1,17 @@
+version: 2
+updates:
+  - package-ecosystem: bundler
+    directory: /
+    schedule:
+      interval: daily
+    allow:
+      # Internal gems
+      - dependency-name: "govuk*"
+        dependency-type: direct
+      # Framework gems
+      - dependency-name: puppet
```

Comment:
Although it is a framework dependency, I don't see us upgrading to a newer version of Puppet any time soon and it would have to be a conscious effort rather than just merging in a Dependabot PR. I reckon to avoid the PRs, it's probably better to remove this from the list.